src/main/groovy/com/bmuschko/gradle/docker/DockerJavaApplication.groovy

```diff
@@ -18,6 +18,17 @@ package com.bmuschko.gradle.docker
 class DockerJavaApplication {
     String baseImage = 'java'
     String maintainer = System.getProperty('user.name')
+    @Deprecated
     Integer port = 8080
+    Set<Integer> ports = []
     String tag
+
+    Integer[] getPorts() {
+        if (ports.size() > 0) {
```

Comment:
Think we can use the ternary operator here to make this more terse `ports ? ports : [port]`